doc/md/fundamentals/2-types/10-options-results.md

```diff
@@ -180,13 +180,10 @@ In the case of a `Result`, you can also use pattern matching with the difference
 
 ```motoko no-repl
 func greetResult(resultName : Result<Text, Text>) : Text {
-  switch (resultName) {
-    case (#err(error)) { "No name: " # error };
-    case (#ok(name)) { "Hello, " # name };
-  }
+  let #ok(name) = resultname else return #err("No Name")
 };
 assert(greetResult(#ok("Motoko")) == "Hello, Motoko!");
-assert(greetResult(#err("404 Not Found")) == "No name: 404 Not Found");
+assert(greetResult(#err("404 Not Found")) == "No name");
 ```
 
```

Comment:
Maybe mention the Result library and say that it provide similar utility functions to Option.get, map etc (without going into details).